src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -370,6 +355,10 @@ export default class StyleConfigurator {
       .join(';');
   }
 
+  computeMessageStyle(edge: Edge): string {
```

Comment:
:question: same remarks as in other place, can we improve the name of this function instead of just `message`?